framework/source/class/qx/lang/Type.js

```diff
@@ -101,7 +101,9 @@ qx.Bootstrap.define("qx.lang.Type",
       // JavaScript null and a null returned from calling DOM.
       // e.q. by document.getElementById("ReturnedNull").
       return (
-        value !== null && (
+        value !== null &&
+        // Check if value is NaN
+        value === value && (
```

Comment:
Shouldn't this contain something like isNaN()? I guess it is a bit more intuitive, or is there a reason to do so?